src/index.html

```diff
@@ -5,8 +5,354 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header class="nav">
+      <nav class="nav__container">
+        <a class="nav__link is-active" href="#">APPLE</a>
+        <a class="nav__link" href="#">samsung</a>
+        <a class="nav__link" href="#">смартфоны и телефоны</a>
+        <a
+          class="nav__link"
+          data-qa="nav-hover"
+          href=""
```

Comment:
```suggestion
          href="#"
```
Не оставляй href пустым